session/bootstrap.go

```diff
@@ -1546,6 +1549,13 @@ func upgradeToVer73(s Session, ver int64) {
 	doReentrantDDL(s, CreateCapturePlanBaselinesBlacklist)
 }
 
+func upgradeToVer74(s Session, ver int64) {
```

Comment:
Do we need to update the bootstrap branch?
When it's bootstrapped from a new cluster is the default value correct?